bookshelf/test/lib/GetConfigTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Bookshelf;
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class GetConfigTrait
+ * @package Google\Cloud\Samples\Bookshelf
+ *
+ * Use this trait to load the project configuration
+ */
+trait GetConfigTrait
+{
+    protected function getConfig()
+    {
+        $config = [];
+
+        // if the settings.yml file exists, copy it over
+        if (file_exists(__DIR__ . '/../../config/settings.yml')) {
```

Comment:
Do you really need to read the configuration file in the test and merge it with the envvars? In my opinion, you can just create an array from the env vars can't you?